BalancedTree/BalancedTree.js

```diff
@@ -0,0 +1,60 @@
+class TreeNode {
+    constructor(val) {
+      this.val = val;
+      this.left = null;
+      this.right = null;
+    }
```

Comment:
Avoid explicitly setting your variables to `null` in JS (that is a very Java thing). JS will automatically set variables to `undefined` if you don't give them values. Also, avoid shortening variable names like this.
```suggestion
    constructor(value) {
      this.value = value;
      this.left;
      this.right;
    }
```